Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
```

Comment:
given this is not strictly useful *inside* the python package, I think this file should be excluded from the MANIFEST (contrary to what `check-manifest` suggests)